cairo-contracts/protobuf/src/types/tag.cairo

```diff
@@ -0,0 +1,60 @@
+#[derive(Copy, Debug, Clone, Drop, PartialEq)]
+pub enum WireType {
+    Varint,
+    Fixed64,
+    LengthDelimited,
+    StartGroup,
+    EndGroup,
+    Fixed32,
+    Unknown,
+}
```

Comment:
Is this variant necessary? Might we ever fall into this?

Reply:
You're right ! I forgot to remove this after correcting a silly mistake on my part.

ed9199c